contracts/src/systems/summit.cairo

```diff
@@ -66,6 +70,22 @@ pub mod summit_systems {
                 // set the new summit beast
                 self._set_summit_beast(new_summit_beast_token_id);
 
+                // set live stats
```

Comment:
lets figure out a way to initialize the summit with a beast in a fair way such that we don't need this special condition in attack